tasks/apm-inject-check.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
I would suggest checking here that `datadog_apm_instrumentation_enabled` is one of `all`, `host`, `docker` and failing early if it is not.

Reply:
added in line 12